ibis-server/dto.py

```diff
@@ -0,0 +1,27 @@
+from pydantic import BaseModel, Field
+
+
+class IbisDTO(BaseModel):
+    sql: str
+    pass
+
+
+class PostgresDTO(IbisDTO):
+    host: str = Field(examples=["localhost"])
+    port: int = Field(default=5432)
+    user: str
+    password: str
+
+
```

Comment:
Just take a note:
I notices that there're different number of empty line between functions or classes. I think we may can find some solution to format the code. To ensure everyone has same coding style.

Reply:
@goldmedal @grieve54706 I recommend using ruff. https://github.com/astral-sh/ruff

Reply:
My default IDE follows the PEP8. It's a regular coding style.

Reply:
> @goldmedal @grieve54706 I recommend using ruff. https://github.com/astral-sh/ruff

Nice idea, I will add it in the future.